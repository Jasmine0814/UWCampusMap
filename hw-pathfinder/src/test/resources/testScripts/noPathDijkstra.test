# no path will be find in this test

CreateGraph Graph1

AddNode Graph1 A
AddNode Graph1 B
AddNode Graph1 C

AddEdge Graph1 A B 1.0

FindPath Graph1 A C
