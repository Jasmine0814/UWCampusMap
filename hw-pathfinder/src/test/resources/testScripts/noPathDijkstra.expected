# no path will be find in this test

created graph Graph1

added node A to Graph1
added node B to Graph1
added node C to Graph1

added edge 1.000 from A to B in Graph1

path from A to C:
no path found
